src/styles/product-card/product-card.css

```diff
@@ -0,0 +1,71 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Georgia, sans-serif;
+  border: 1px solid rgba(243, 243, 243, 1);
+  outline-offset: 1px;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__title {
+  margin-top: 40px;
```

Comment:
add margin-bottom to img instead top for title

Reply:
fixed